### ISSUE TITLE
[Bug]: File upload > 30mb resulting in renderer crash, when using 32bit electron on 64bit windows

### ISSUE THREAD

**Issue by roopaeyes** [electron/electron#35631]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/main/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a bug report that matches the one I want to file, without success.

### Electron Version

20.0.3

### What operating system are you using?

Windows

### Operating System Version

Windows 10

### What arch are you using?

x64

### Expected Behavior

Expecting the same beaviour as 64 bit  i, e allow upload of any file size without render crash

### Actual Behavior

Hi,

My applicaiton is using angularjs, Node and electron.
when we try to upload any file of size greater than 30mb on an App built on 32 bit ("electron-builder --win --ia32",), the following exception is thrown
![render](https://user-images.githubusercontent.com/13520145/189431424-63cfb19b-c864-4aea-a366-4783c71fcdf5.PNG)

However it works fine on app built on 64 bit("electron-builder --win --x64").

index.html
<html>
   <head>
      <script src = "https://ajax.googleapis.com/ajax/libs/angularjs/1.3.14/angular.min.js"> </script>
       <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
       <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.0/jquery.min.js"></script>
       <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
       <script type="text/javascript" src="/public/angular-base64-upload.js"></script>
   </head>
   
   <body ng-app = "myApp">	
      <div ng-controller = "myCtrl">
         <input type='file'  ng-model='myModel' on-after-validate = "onAfterValidateFunc" multiple ng-model="files" name="files" id="uploaddocs" base-sixty-four-input >
         <br /><br />  
            Status: {{ Status }} 
      </div>
      
      <script> 
       var myApp = angular.module('myApp',['naif.base64']);  
       myApp.controller('myCtrl', function($scope,$q) {
        $scope.Status = "Not started";      
         $scope.onAfterValidateFunc = function(e, fileObjects,fileList){
           $scope.Status = "Done";
        }        
      });
      </script>
      
   </body>
</html>

********************************
app.js

var express = require('express');
var bodyParser = require('body-parser');
var http = require('http');
var httpApp = express();
httpApp.use("/public",express.static("public"));

httpApp.get('/index', function (req, res) {   
     res.sendFile(__dirname + "/"+ "index.html");
 })
var httpServer = httpApp.listen(8080,'localhost', function(){
    console.log("angular hosted server started ")
});

**********************************************
main.js
const { app, BrowserWindow,dialog } = require('electron')
app.commandLine.appendSwitch('ignore-certificate-errors');
app.commandLine.appendSwitch('allow-insecure-localhost',true);

var ipc = require('electron').ipcRenderer;

function createWindow () {
  win = new BrowserWindow({
    width: 800,
    height: 600,
    icon: ( './icon.png'),
    webPreferences: {
      devTools: true
    }
  })
  win.maximize();
  win.removeMenu();
  win.webContents.openDevTools();
 
  win.loadURL('http://localhost:8080/index')

  win.on('closed', () => { 
    win = null
  })
}
 
app.whenReady().then(()=>{
  app.disableHardwareAcceleration();
  createWindow();
  app.on('activate', function(){
    if(BrowserWindow.getAllWindows().length === 0){
      createWindow();
    }
  });
});

app.on('ready', createWindow)

app.on('window-all-closed', () => {
  if (process.platform !== 'darwin') {
    app.quit()
  }
})
 
app.on('activate', () => {
  if (win === null) {
    createWindow()
  }
})

app.on("browser-window-created", (e,win)=>{
e.preventDefault();
win.removeMenu();
win.maximize();
});

Kindly help

### Additional Information
Git hub link : https://github.com/roopaeyes/ElectronFileUploadSample
_No response_

**Comment by VerteDinde** [comment]
Thanks for reporting this issue! Would it be possible for you to make a standalone testcase with only the code necessary to reproduce the issue? For example, [Electron Fiddle](https://www.electronjs.org/fiddle) is a great tool for making small test cases and makes it easy to publish your test case to a gist that Electron maintainers can use.

Stand-alone test cases make fixing issues go more smoothly: it ensure everyone's looking at the same issue, it removes all unnecessary variables from the equation, and it can also provide the basis for automated regression tests.

I'm adding the blocked/need-repro label for this reason. After you make a test case, please link to it in a followup comment.

Thanks in advance!

**Comment by codebytere** [comment]
We haven't gotten a response to our questions in our comment above. With only the information that is currently in the issue, we don't have enough information to take action. In this event, i'm going to go ahead and close this but can reopen should you follow up with more info!
